lib/hyacinth/digital_object/type_def/json_serializable_collection.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Hyacinth
+  module DigitalObject
+    module TypeDef
+      class JsonSerializableCollection < Hyacinth::DigitalObject::TypeDef::Base
+        def initialize(type = nil)
+          super()
```

Comment:
```suggestion
          super
```

Reply:
This would cause an error because the super method has no arguments defined, and the bar call to super passes all the arguments the method received.

Reply:
Ohhhh, right.